packages/components/tests/dummy/app/templates/components/tooltip.hbs

```diff
@@ -0,0 +1,206 @@
+      >
+        <FlightIcon @name="alert-circle" />
+      </Hds::TooltipButton>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H4>Used <em>incorrectly</em> to wrap a block-level element (button wraps div tag)</Shw::Text::H4>
+  <Shw::Text::Body>(results in invalid HTML structure)</Shw::Text::Body>
+  <Hds::TooltipButton @text="Invalid usage">
+    <div><FlightIcon @name="alert-circle" /></div>
+  </Hds::TooltipButton>
+
+  <Shw::Text::H4>Used next to text</Shw::Text::H4>
+  <p>
+    <strong>Lorem ipsum dolor sit amet consectetur</strong>
+    <Hds::TooltipButton @text="Hello!"><FlightIcon @name="alert-circle" /></Hds::TooltipButton>
+  </p>
+
+  <Shw::Text::H4>Used within a sentence.</Shw::Text::H4>
+  <p>
+    Lorem ipsum dolor sit amet consectetur
+    <Hds::TooltipButton @text="Hello!"><FlightIcon @name="alert-circle" /></Hds::TooltipButton>
+    adipisicing elit. Doloremque blanditiis sapiente iste beatae voluptates voluptatum.
+  </p>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H3>States</Shw::Text::H3>
+
+  <Shw::Text::Body>
+    The button which serves as the opener for the tooltip has no special styles of its own other than for the focus
+    state.
+  </Shw::Text::Body>
+
+  <Shw::Flex as |SF|>
+    {{#let (array "default" "focus") as |states|}}
+      {{#each states as |state|}}
+        <SF.Item @label={{state}}>
+          <div class="shw-component-side-nav-home-link-wrapper">
+            <Hds::TooltipButton @text="Hello!" mock-state-value={{state}}><FlightIcon
+                @name="alert-circle"
+              /></Hds::TooltipButton>
+          </div>
+        </SF.Item>
+      {{/each}}
+    {{/let}}
+
+    <SF.Item @label="Tooltip which appears on hover or focus">
+      {{! template-lint-disable no-inline-styles }}
+      <div data-tippy-root="" id="tippy-9" style="position: relative;">
+        <div
+          class="tippy-box"
+          data-state="visible"
+          tabindex="-1"
+          data-theme="hds"
+          data-animation="fade"
+          role="tooltip"
+          data-placement="top"
+          style="width: min-content;"
+        ><div class="tippy-content" data-state="visible">Hello!</div><div
+            class="tippy-svg-arrow"
+            style="position: absolute; left: 0px; transform: translate3d(22.5px, 0px, 0px);"
+          >
+            <svg
+              class="hds-tooltip-pointer"
+              width="16"
+              height="7"
+              viewBox="0 0 16 7"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M0 7H16L9.11989 0.444571C8.49776 -0.148191 7.50224 -0.148191 6.88011 0.444572L0 7Z"></path>
+            </svg></div></div></div>
+      {{! template-lint-enable no-inline-styles }}
```

Comment:
The display of the tooltip is controlled by Tippy.js JavaScript code. The only way I could think to include it for the examples was to copy and paste the tooltip into the DOM.

I don't actually think it's a good idea to add this static copy and pasted code for the tooltip like this but I thought I'd just include it to get feedback on other possible ideas...